metalearn/metafeatures/landmarking_metafeatures.py

```diff
@@ -26,17 +30,40 @@ def run_pipeline(X, Y, pipeline, n_folds, cv_seed):
     with warnings.catch_warnings():
         warnings.filterwarnings("ignore", category=RuntimeWarning)  # suppress sklearn warnings
         warnings.filterwarnings("ignore", category=UserWarning)  # suppress sklearn warnings
-        accuracy_scorer = make_scorer(accuracy_score)
-        kappa_scorer = make_scorer(cohen_kappa_score)
         cv = StratifiedKFold(n_splits=n_folds, shuffle=True, random_state=cv_seed)
-        scores = cross_validate(
```

Comment:
All the code in `run_pipeline` should not change. Your new cross validation function should replace the old one.